turbopack/crates/turbo-tasks-backend/src/data.rs

```diff
@@ -534,9 +536,9 @@ transient_traits!(InProgressCellState);
 impl Eq for InProgressCellState {}
 
 impl InProgressCellState {
-    pub fn new(task_id: TaskId, cell: CellId) -> Self {
+    pub fn new(_task_id: TaskId, _cell: CellId) -> Self {
         InProgressCellState {
-            event: Event::new(move || format!("InProgressCellState::event ({task_id} {cell:?})")),
+            event: new_event!(move || format!("InProgressCellState::event ({_task_id} {_cell:?})")),
```

Comment:
Shouldn't the `if false` branch prevent you from needing the underscore prefixes to variable names here? (Same goes for other callsites)